### PR TITLE
PostTypeList: add PostActions to PostItem

### PR DESCRIPTION
**Pull request by jonathansadowski**
**This PR is in progress, do not merge.**

This PR adds `PostActions` to `PostItem` to be displayed in `PostTypeList`.

### CODE REVIEW

File: client/blocks/post-item/index.jsx [hunk 1]
```diff
@@ -145,19 +146,27 @@ class PostItem extends React.Component {
...
+									>
+										{ title || translate( 'Untitled' ) }
+									</ExternalLink>
+								) }
 						</h1>
 						<div className="post-item__meta">
 							<PostTime globalId={ globalId } />
+							{ post && (
+								<PostActions
+									className={ 'post-item__post-actions' }
```
Comment (nylen) [inline client/blocks/post-item/index.jsx]: We can just use `""` instead of `{ '' }` here.

Comment (jonathansadowski) [reply client/blocks/post-item/index.jsx]: 👍 Updated.

Comment (jonathansadowski) [reply client/blocks/post-item/index.jsx]: It appears they are actually being changed by prettifier when I attempt to commit.

Comment (nylen) [reply client/blocks/post-item/index.jsx]: Ugh.

Comment (jonathansadowski) [reply client/blocks/post-item/index.jsx]: Ignore me -- I'm having problems parsing English this morning :)

**Comment by apeatling** [comment]
@jonathansadowski Still in progress?

**Comment by jonathansadowski** [comment]
Thanks for the ping. It is not. This branch is no longer needed.

**Pull request closed without merging**